### PR TITLE
Move VACUUM FULL test cases from TINC.

### PR DESCRIPTION
**Pull request by hlinnaka**
We have some VACUUM FULL commands in other tests already, but these tests
seem more comprehensive than the other tests.

### CODE REVIEW

**Comment by Eulerizeit** [comment]
Does this or will this include removing the others?

Rob

**Comment by hlinnaka** [comment]
On 02/20/2017 07:06 PM, Robert Eckhardt wrote:
> Does this or will this include removing the others?

No. The other VACUUM FULL commands are sprinkled around various tests. 
Might be a good idea to review them, to see if any of them could be 
removed as redundant, but I wasn't planning to do that right now.

- Heikki

File: src/test/regress/sql/vacuum_full_ao.sql [hunk 1]
```diff
@@ -1,6 +1,3 @@
--- start_ignore
-SET gp_create_table_random_default_distribution=off;
```
Comment (ashwinstar) [inline src/test/regress/sql/vacuum_full_ao.sql]: Does removing this invalidates the tests with ORCA runs ? Should be modify Create statements to explicitly include `DISTRIBUTED BY` then if removing this GUC.

File: src/test/regress/sql/vacuum_full_freeze_heap.sql [hunk 2]
```diff
@@ -1,6 +1,3 @@
--- start_ignore
-SET gp_create_table_random_default_distribution=off;
--- end_ignore
 drop table if exists vfheap;
```
Comment (ashwinstar) [inline src/test/regress/sql/vacuum_full_freeze_heap.sql]: Lets start loosing out these `drop table if exists` statements.

**Comment by hlinnaka** [comment]
On 02/21/2017 09:56 AM, Ashwin Agrawal wrote:
> ashwinstar approved this pull request.
>
> 👍
>
>> @@ -1,6 +1,3 @@
> --- start_ignore
> -SET gp_create_table_random_default_distribution=off;
>
> Does removing this invalidates the tests with ORCA runs ? Should be modify Create statements to explicitly include `DISTRIBUTED BY` then if removing this GUC.

No, the GUC defaults to 'off', anyway. ORCA doesn't have a say in the 
chosen distribution. (Except with CREATE TABLE AS SELECT. ORCA behaves 
differently on that, but that's irrespective of 
gp_create_table_random_default_distribution)

>> @@ -1,6 +1,3 @@
> --- start_ignore
> -SET gp_create_table_random_default_distribution=off;
> --- end_ignore
>  drop table if exists vfheap;
>
> Lets start loosing out these `drop table if exists` statements.

Agreed. I was lazy.

- Heikki

**Comment by hlinnaka** [comment]
Pushed, thanks for the review. I left the DROP TABLE IF EXISTS commands in place, after all, because these tests actually use the same table names, so just by removing the DROP TABLE commands, they would conflict. I could've renamed the test tables, or created a unique schema for each test, but it's not that important for these tests, as they cannot run in parallel, anyway.

**Pull request closed without merging**